blocks/file-blocks/markdown-edit/extensions.ts

```diff
@@ -124,6 +126,31 @@ export function makeExtensions({
       }
     }),
 
+    EditorView.domEventHandlers({
+      paste(e, view) {
+        const value = e.clipboardData?.items[0];
+        if (value.type === "image/png") {
```

Comment:
seems like you could handle other image types here without any other changes

Reply:
thanks for calling me on my laziness - I wasn't sure how to test with a jpg, but added support for this & "copy image" from a website as well